README.md

```diff
@@ -30,7 +30,9 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
 
 2. Fork the project, using the top right corner button, that states `fork`
 
-3. In the `data.json` file, add your own object to the **end of the file**. Your entry should be similar to this example:
+3. Create a file named `your-username.json` in the directory [`profiles`](/profiles) with the following content. 
```

Comment:
This link won't work properly. Use `tree/main/profiles` instead (if you wanna use a relative link).
```suggestion
3. Create a file named `your-username.json` in the directory [`profiles`]/(tree/main/profiles) with the following content. 
```

Reply:
### Thanks for the Suggestion,
- **But the Suggested format is incorrect, forward slash `/` should be inside small brackets`( )`.**

**Screenshot** If we use the suggested change

---

![Screenshot from 2021-12-05 21-29-44](https://user-images.githubusercontent.com/51878265/144753910-5e16bc17-9ff5-4495-b803-ddd3c7e63a40.png)

---

- **The correct format is: (which i used)**
```sh 
[a relative link](/dir name)
```
> Also, you can check official [docs](https://www.markdownguide.org/basic-syntax/#links) 

- **We don't need to add any pre-path like `tree/main`, we can Directly put the `Directory` name, it will work 100/100, times that the how Markdown functions and its specialty.**

Reply:
Ah yes, I made a mistake here 😅 Fixed now.

Reply:
It will not work you can test it.